.template/addons/ecs_deploy/deploy_staging_ecs.yml.tt

```diff
@@ -0,0 +1,47 @@
+name: Deploy to ECS staging
+
+on:
+  workflow_run:
+    workflows:
+      - Test
+    branches:
+      - develop
+    types:
+      - completed
+  workflow_dispatch:
+
+env:
+  DOCKER_REGISTRY_HOST: ${{ secrets.ECR_REGISTRY_HOST }}
+  DOCKER_IMAGE: ${{ secrets.STAGING_ECR_REPO }}
+  ECS_CLUSTER: ${{ secrets.STAGING_ECS_CLUSTER }}
+  ECS_SERVICE: ${{ secrets.STAGING_ECS_SERVICE }}
+  BRANCH_TAG: latest
+
+jobs:
+  deploy:
+    name: Deploy to ECS staging
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cancel previous runs
+        uses: styfle/cancel-workflow-action@0.6.0
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.AWS_REGION }}
+
+      - uses: actions/checkout@v2.3.4
+
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+
+      - name: Build Docker image
+        run: bin/docker-prepare && docker-compose build
+
+      - name: Push to ECR
```

Comment:
Maybe we can add the deployment command as well, although it can be tested later when infra is ready on my end.